cli/tests/integration/npm_tests.rs

```diff
@@ -2483,6 +2483,62 @@ console.log(add(1, 2));
   output.assert_matches_text("Check file:///[WILDCARD]/project-b/main.ts\n");
 }
 
+#[test]
+pub fn byonm_cjs_export_analysis_require_re_export() {
+  let test_context = TestContextBuilder::for_npm().use_temp_cwd().build();
+  let dir = test_context.temp_dir();
+  dir.write(
+    "deno.json",
+    r#"{
+    "unstable": [
+      "byonm"
+    ]
+  }"#,
+  );
+
+  dir.write(
+    "package.json",
+    r#"{
+  "name": "my-package",
```

Comment:
I'm a bit confused - wasn't the problem that the `name` field contained something like `package/name`?

Reply:
I messed up writing the package's package.json, but it actually does still test this. I will fix it soon

Reply:
This is fixed and improved now. Actually a confusing part here was that this package had the same name as the package it was importing 😅